src/events/ready.ts

```diff
@@ -62,33 +63,14 @@ async function registerRolesChannels(client: DatadropClient): Promise<void> {
   ]);
 }
 
-async function handleDynamicChannels(client: DatadropClient, log: Logger) {
-  const { dynamicChannelPrefix } = config;
-  client.channels.cache
-    .filter((c: Channel) => c.type === ChannelType.GuildVoice && c.name.includes(dynamicChannelPrefix))
-    .map((c: Channel) => c as VoiceChannel)
-    .forEach((c: VoiceChannel) => {
-      const author = c.permissionOverwrites.cache.find(po => po.type === OverwriteType.Member);
-      if (author) {
-        client.dynamicChannels.set(c.id, {
-          authorId: author.id,
-          voiceChannel: c,
-          textChannel: c.parent!.children.cache.find(c => c.type === ChannelType.GuildText && c.permissionOverwrites.cache.some(po => po.type === OverwriteType.Member && po.id === author.id))
-        });
-      }
-    });
-
-  client.dynamicChannels = client.dynamicChannels
-    .each(async dChannelInfo => {
-      if (dChannelInfo.voiceChannel.members.size === 0) {
-        await dChannelInfo.voiceChannel.delete();
-        log.info(`Plus d'utilisateurs dans <${dChannelInfo.voiceChannel.name}> (${dChannelInfo.voiceChannel.id}). Canal supprimé.`);
-
-        if (dChannelInfo.textChannel) {
-          await dChannelInfo.textChannel.delete();
-          log.info(`Canal vocal temporaire supprimé, canal écrit <${dChannelInfo.textChannel.name}> (${dChannelInfo.textChannel.id}) également supprimé.`);
-        }
-      }
-    })
-    .filter(dChannelInfo => dChannelInfo.voiceChannel.members.size > 0);
+async function registerDynamicChannels(client: DatadropClient, config: Configuration): Promise<void> {
```

Comment:
No need to have the `config` parameter as it is already present as a constant at line 6 😄